web/src/app/layout.css

```diff
@@ -1,10 +1,24 @@
-.layout {
+.content__wrapper {
   display: flex;
   min-height: 100vh;
   font-family: sans-serif;
 }
 
-.main-content {
+main.content {
   flex: 1;
   padding: 2rem;
 }
+
+.page {
+  height: 100svh;
+  justify-items: center;
+}
```

Comment:
It looks like the `layout.tsx` does not contain the `page` class?

Reply:
It depends if you are using the `page` class across multiple components? If it is designed to be reusable then it could go in `globals.css`. But my gut feeling tells me it's not really necessary, at the moment the `.page` class feels more like a layout style. in which case it might be better combined with `.layout`